x-pack/plugins/synthetics/common/rules/alert_actions.test.ts

```diff
@@ -60,6 +61,70 @@ describe('Legacy Alert Actions factory', () => {
     ]);
   });
 
+  it('generate expected action for email', async () => {
+    const resp = populateAlertActions({
+      groupId: MONITOR_STATUS.id,
+      defaultActions: [
+        {
+          actionTypeId: '.email',
+          group: 'xpack.uptime.alerts.actionGroups.monitorStatus',
+          params: {
+            dedupKey: 'always-downxpack.uptime.alerts.actionGroups.monitorStatus',
+            eventAction: 'trigger',
+            severity: 'error',
+            summary: MonitorStatusTranslations.defaultActionMessage,
+          },
+          id: 'f2a3b195-ed76-499a-805d-82d24d4eeba9',
+        },
+      ] as unknown as ActionConnector[],
+      translations: {
+        defaultActionMessage: MonitorStatusTranslations.defaultActionMessage,
+        defaultRecoveryMessage: MonitorStatusTranslations.defaultRecoveryMessage,
+        defaultSubjectMessage: MonitorStatusTranslations.defaultSubjectMessage,
+        defaultRecoverySubjectMessage: MonitorStatusTranslations.defaultRecoverySubjectMessage,
+      },
+      isLegacy: true,
+      defaultEmail: {
+        to: ['test@email.com'],
+      },
+    });
+    expect(resp).toEqual([
+      {
+        group: 'recovered',
+        id: 'f2a3b195-ed76-499a-805d-82d24d4eeba9',
+        params: {
+          bcc: [],
+          cc: [],
+          kibanaFooterLink: {
+            path: '',
+            text: '',
+          },
+          message:
+            'Alert for monitor {{context.monitorName}} with url {{{context.monitorUrl}}} from {{context.observerLocation}} has recovered',
```

Comment:
```suggestion
            'The monitor {{context.monitorName}} checking {{{context.monitorUrl}}} from {{context.observerLocation}} has recovered. The alert is no longer active.',
```
Trying to be more conversational, feel free to ignore if not relevant here.